### DISCUSSION TITLE
Distiller updates

### DISCUSSION THREAD

**Discussion by agalitsyna** [open2c/distiller-nf#170] [Ideas]
A couple of points on distiller that might improve the user experience with it:
1. Distiller needs docs. Parameters of the project.yml file can be described in nice documentation instead of comment lines. 
2. There is a special tool for [building the docs from the JSON schema in nf-core](https://nf-co.re/tools/#pipeline-schema). It might be used without complying with other nf-core rules. Do we probably want to try it? 
3. @Phlya pointed to the lengthy commands in the distiller processes, and it might be solved by nextflow's [stdout output of the channels.](https://www.nextflow.io/docs/latest/process.html#outputs) 
4. We might consider transferring to DSL2, but first, we probably want to discuss the modules/subworkflows/workflows structure.

**Comment by golobor** [comment]
+1 to all suggestions - docs/compliance with nf-core standards/DSL2!
